### DISCUSSION TITLE
Option to "hide" Shop Packs from Website Display vrs In-Game Display

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#400] [Suggestions]
Pretty much what it says on the tin. The Web Shop UI is very handy for creating fake shop packs that contain useful information for players, like rules or plugin commands. Since these would be far more relevant in-game than in the web-shop, it'd be useful if there was a toggle to prevent them from showing up in the web shop display, and vice-versa.

**Comment by ad3d0** [comment]
I agree, such option would be handy. We run several different games in our community, so would be nice if we had an option to hide non ARK related shop packs from in-game store.
